.github/workflows/trigger-e2e-tests.yml

```diff
@@ -0,0 +1,194 @@
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "compute-node-v14"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+
+      - name: check if vm-compute-node-v14 image is build
+        if: ${{ steps.check-compute-node-v14-ecr-image.outputs.image-exists == '0' }}
+        id: check-vm-compute-node-v14-ecr-image
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "vm-compute-node-v14"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+
+      - name: check if compute-node-v15 image is build
+        if: ${{ steps.check-vm-compute-node-v14-ecr-image.outputs.image-exists == '0' }}
+        id: check-compute-node-v15-ecr-image
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "compute-node-v15"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+
+      - name: check if vm-compute-node-v15 image is build
+        if: ${{ steps.check-compute-node-v15-ecr-image.outputs.image-exists == '0' }}
+        id: check-vm-compute-node-v15-ecr-image
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "vm-compute-node-v15"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+
+      - name: check if compute-node-v16 image is build
+        if: ${{ steps.check-vm-compute-node-v15-ecr-image.outputs.image-exists == '0' }}
+        id: check-compute-node-v16-ecr-image
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "compute-node-v16"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
+
+      - name: check if vm-compute-node-v16 image is build
+        if: ${{ steps.check-compute-node-v16-ecr-image.outputs.image-exists == '0' }}
+        id: check-vm-compute-node-v16-ecr-image
+        uses: mnmandahalf/check-ecr-image-exists@v0.1.5
+        with:
+          region: eu-central-1
+          repository-name: "vm-compute-node-v16"
+          image-tag: "${{ github.event.inputs.build-tag }}"
+          access-key-id: ${{ env.AWS_ACCESS_KEY_ID }}
+          secret-access-key: ${{ env.AWS_SECRET_ACCESS_KEY }}
```

Comment:
Is it necessary to introduce a separate action for it?
Can we use `aws-cli`?
```
aws ecr describe-images --repository-name ${REPO} --region eu-central-1 --query "imageDetails[?imageTags[?contains(@, '${TAG}')]]" 
```
If the command return empty json array (`[]`), then the image doesn't exist

Reply:
Done